3DAmsterdam/Assets/Amsterdam3D/Scripts/LayerSystem/TileHandler.cs

```diff
@@ -297,11 +301,9 @@ private GameObject BuildNewTile(AssetBundle assetBundle, TileChange tileChange)
             //positioning container
             Vector3RD hoekpunt = new Vector3RD(X, Y, 0);
```

Comment:
Variables horen in het Engels te zijn

Reply:
Inderdaad. Is een legacy scriptje; maar zal hem wel even netjes/engels maken 👍